.github/workflows/pr-builder.yml

```
@@ -21,6 +21,58 @@ env:
+    strategy:
+      matrix:
+        node-version: [ lts/* ]
+        pnpm-version: [ latest ]
+    steps:
+      - name: ⬇️ Checkout
+        id: checkout
+        uses: actions/checkout@v2.3.3
+
+      - name: 🟢 Setup node
+        id: setup-node
+        uses: actions/setup-node@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)